kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -507,6 +507,15 @@ object KyuubiConf {
       .serverOnly
       .fallbackConf(FRONTEND_BIND_PORT)
 
+  val FRONTEND_THRIFT_CONNECTION_URL_HOST: ConfigEntry[String] =
+    buildConf("kyuubi.frontend.thrift.connection.url.host")
+      .doc("Hostname or IP of the Kyuubi server's thrift frontend service that will be " +
+        "registered for the service discovery ensemble.")
+      .version("1.8.0")
+      .serverOnly
+      .stringConf
+      .createWithDefault("")
```

Comment:
how about using `OptionalConfigEntry` and `createOptional` instead?

Reply:
Fixed at 34c61d9.